utils/display/resources_test.go

```diff
@@ -126,6 +126,21 @@ var types = []meroxa.ResourceType{
 			meroxa.ResourceTypeFormConfigHumanReadableKey: "PostgreSQL",
 		},
 	},
+	{
+		Name:         string(meroxa.ResourceTypeGoogleSheets),
+		ReleaseStage: meroxa.ResourceTypeReleaseStageDevPreview,
+		HasAccess:    true,
+		FormConfig: map[string]interface{}{
+			meroxa.ResourceTypeFormConfigHumanReadableKey: "Google Sheets",
+		},
+	},
+	{
+		Name:         string(meroxa.ResourceTypeGoogleAnalytics),
+		ReleaseStage: meroxa.ResourceTypeReleaseStageDevPreview,
+		FormConfig: map[string]interface{}{
+			meroxa.ResourceTypeFormConfigHumanReadableKey: "Googs",
```

Comment:
ResourceTypeGoogleAnalytics == "Googs"?

Reply:
This is just a human readable string that is created to assert that the tests below (`TestResourceTypesTable()` and `TestResourceTypesTableWithoutHeaders()` reflect the same key that's set here. `Googs` is nowhere in our migration files

Reply:
writing "googs" was just me being lazy for a unit test case which was meant to be ignored on success